providers/build.info

```diff
@@ -1 +1,13 @@
 SUBDIRS=common default
+
+IF[{- !$disabled{fips} -}]
+  SUBDIRS=fips
+  MODULES=fips
+  SHARED_SOURCE[fips]=fips.ld
+  GENERATE[fips.ld]=provider.num
```

Comment:
These kinds of lines is usually wrapped like this:

```
  IF[{- defined $target{shared_defflag} -}]
    SHARED_SOURCE[fips]=fips.ld
    GENERATE[fips.ld]=provider.num
  ENDIF
```

That's easily forgotten, though, so maybe Configure should be adjusted to ignore any `.ld` file spec if `$target{shared_defflag}` isn't defined...  it seems a bit "magic" for my taste, but...

Reply:
The lack of this guard is what causes the `Unknown operating system family darwin` issue.

Reply:
Ah, thanks! Will fix. That was non-obvious!

Reply:
Fixup pushed. Lets see if travis is happier.